src/main/java/org/matsim/analysis/preAnalysis/PopulationAnalysis.java

```diff
@@ -0,0 +1,92 @@
+package org.matsim.analysis.preAnalysis;
+
+import org.locationtech.jts.geom.Geometry;
+import org.matsim.api.core.v01.Coord;
+import org.matsim.api.core.v01.Scenario;
+import org.matsim.api.core.v01.population.Activity;
+import org.matsim.api.core.v01.population.Person;
+import org.matsim.api.core.v01.population.PlanElement;
+import org.matsim.api.core.v01.population.Population;
+import org.matsim.application.MATSimAppCommand;
+import org.matsim.core.config.Config;
+import org.matsim.core.config.ConfigUtils;
+import org.matsim.core.scenario.ScenarioUtils;
+import org.matsim.core.utils.gis.ShapeFileReader;
+import org.opengis.feature.simple.SimpleFeature;
+import picocli.CommandLine;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Collection;
+
+public class PopulationAnalysis implements MATSimAppCommand {
+    public static void main(String[] args) throws IOException {
+        new PopulationAnalysis().execute(args);
+    }
+
+    @CommandLine.Option(names= "--population", description = "Path to input population", required = true)
+    private String populationPath;
+
+    @CommandLine.Option(names= "--output", description = "Path to analysis output", required = true)
+    private String outputPath;
+
+    @Override
+    public Integer call() throws Exception {
+        String shapeFilePath = "./scenarios/input/shp/dilutionArea.shp";
```

Comment:
You can use ShpOptions for this.

Reply:
The shapefile needs to be in the repository for the testing purpose. So I think it is good to just use it.

Reply:
But anyway, I have switched to the shp option. In that case, the script can be used by other scenario as well.

Reply:
normally, we have a rather strong guideline not to put (large) data into github repos, in order not to blow the history up. it would be better to put such data on public svn. maybe there is still a way to do that and keep the history clean?

Reply:
The file is not very large, only 26kb. If you remove it and squash the commit, it should not be in the history.

Reply:
Currently our shp option cannot read from the SVN...The Shapefile reading tool can also not read it properly. We will need to update it in the matsim core or update the shp option. 
Below is the error message:
Exception in thread "main" java.lang.IllegalStateException: Shape file https:/svn.vsp.tu-berlin.de/repos/public-svn/matsim/scenarios/countries/de/kelheim/shp/dilutionArea.shp does not exists

Reply:
You could add some code to download the shp files locally into the folder where you had it before, if it is not present.